packages/api/package.json

```diff
@@ -24,6 +24,7 @@
     "config": "^3.3.1",
     "graphql": "^15.0.0",
     "graphql-tools": "^5.0.0",
+    "mongodb": "^3.5.6",
```

Comment:
Can you add this as a dev dependency instead? We will be using postgres in production :)